CMakeLists.txt

```diff
@@ -395,6 +395,11 @@ if(WITH_LIBCEPHFS OR WITH_KRBD)
   find_package(keyutils REQUIRED)
 endif()
 
+if(WITH_LIBCEPHFS)
+  find_package(PkgConfig REQUIRED)
```

Comment:
nit, could add `QUIET` before `REQUIRED`. as what we are interested in is libcap-ng. like
```cmake
find_package(PkgConfig QUIET REQUIRED)
```

also, i'd suggest move this check into https://github.com/ceph/ceph/blob/e909031d3c231052fdf0afe993f669514faadde6/src/CMakeLists.txt#L641 . i.e., closer to where it is used.